include/dxc/Support/HLSLOptions.td

```diff
@@ -170,6 +170,11 @@ def opt_enable : Separate<["-", "/"], "opt-enable">, Group<hlsloptz_Group>, Flag
 def opt_select : MultiArg<["-", "/"], "opt-select", 2>, MetaVarName<"<opt> <variant>">, Group<hlsloptz_Group>, Flags<[CoreOption, DriverOption, HelpHidden]>,
   HelpText<"Select this optimization variant.">;
 
+def CommaJoinTest : CommaJoined<["-"], "CommaJoinTest=">, Group<spirv_Group>, Flags<[CoreOption, HelpHidden]>,
```

Comment:
I assume these options aren’t actually SPIR-V specific right? They should probably be in a different Group,